ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/sorter/AbstractPendingTransactionsSorter.java

```diff
@@ -443,7 +491,7 @@ public List<Transaction> signalInvalidAndGetDependentTransactions(final Transact
               pendingTx ->
                   traceLambda(
                       LOG,
-                      "Transaction {} piked for removal since there is a lowest invalid nonce {} for the sender",
+                      "Transaction {} piked since there is a lowest invalid nonce {} for the sender",
```

Comment:
Transaction {} invalid since there is a lower invalid nonce {} for the sender

?

Reply:
suggestion applied, tnx